pyproject.toml

```diff
@@ -47,7 +47,6 @@ disable= [
     "too-many-positional-arguments",
     "too-many-branches",
     "too-many-instance-attributes",
-    "unspecified-encoding",
```

Comment:
Without this removal, tests fail with

```
>               assert_that(len(diagnostics), is_(greater_than(0)))
E               AssertionError: 
E               Expected: a value greater than <0>
E                    but: was <0>

src/test/python_tests/test_code_actions.py:406: AssertionError
------------------------------------------------------------------------------------------ Captured log call ------------------------------------------------------------------------------------------
ERROR    concurrent.futures:_base.py:344 exception calling callback for <Future at 0x729661915600 state=finished returned NoneType>
```

Reply:
Also, is there a way to easily test changes "live"?

I didn't see anything discussing that at https://github.com/microsoft/vscode-pylint/wiki/Contributing-Guide.